code/style.css

```diff
@@ -1,5 +1,123 @@
 /* If you can see a peach background, the css file is correctly linked. Feel free to change or remove this styling! */
 
+
 body {
-  background-color: peachpuff;
+  background-color: rgb(180,132,132);
+  display: flex;
+  flex-flow: column wrap;
+  align-items: center;
+}
+
+header {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  flex-grow: 1;
+}
+
+H1 {
+  text-align: center;
+  color: rgb(92,23,30);
+  font-weight: bolder;
+  background-color: antiquewhite;
+  padding: 50px;
+  border-color: rgb(92,23,30);
+  border-style: solid;
+  flex-grow: 1; 
+}
+
+
+.headerimage { max-width: 100%;
+  max-height: 100vh;
+  align-content: center;
+}
+
+
+main {
+  flex-grow: 1;
+  display: flex;
+  flex direction: row;
+  flex-wrap: wrap;
+  justify-content: space-around;
+}
+
+h2 {
+  font-weight: bold; 
+  text-align: center;
+  color: rgb(92,23,30);
+  margin: 20px;  
+}
+
+ul {
+  margin-left: 2%; 
+  padding-left: 2%;
 }
+
+.researchcard {
+  flex-basis: 0;
+  flex-grow: 1;
+  text-align: left;
+  padding-left: 2%;
+  padding-right: 2%;
+  padding-bottom: 2%;
```

Comment:
This could be written like:
`padding: initial 2% 2% 2%`
where we set top padding to initial and then right, bottom, left to 2%